### ISSUE TITLE
Internal server error while exporting Leanpub book to an ePub

### ISSUE THREAD

**Issue by fenuks** [wallabag/wallabag#5061]
Version:  one on wallabag.it

### What steps will reproduce the bug?
Add https://leanpub.com/haskell-cookbook/read, try to export as ePub. See 500 error:
```
Call to a member function getElementsByTagName() on null
```

Exporting to mobi or PDF does work.

**Comment by harmtemolder** [comment]
@fenuks Do you still have this error? I've tried to reproduce it running locally off of `master`, as well as on my live instance (`2.6.10`), but both export this epub without any problems. Maybe this has been solved already?

**Comment by fenuks** [comment]
Yes, it is fixed. Works on current version hosted on wallabag.it (2.6.10).
